data/src/main/java/com/pocs/data/paging/AdminPostByUserPagingSource.kt

```diff
@@ -16,20 +16,19 @@ class AdminPostByUserPagingSource @Inject constructor(
 
     companion object {
         private const val START_PAGE = 1
+        private const val PAGING_SIZE = 30
     }
 
     override suspend fun load(params: LoadParams<Int>): LoadResult<Int, Post> {
         val page = params.key ?: START_PAGE
         return try {
-            val response = api.getAllPostsByUser(userId)
+            val response = api.getAllPostsByUser(userId, PAGING_SIZE, page = page)
```

Comment:
아래처럼 argument가 많은 경우 named argument를 쓰는게 좋아보이네요.
```suggestion
            val response = api.getAllPostsByUser(userId = userId, pageSize = PAGING_SIZE, page = page)
```